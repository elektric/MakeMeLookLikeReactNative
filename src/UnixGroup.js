import React, { Component } from 'react';
import {
  AppRegistry,
  StyleSheet,
  Text,
  View,
  Image
} from 'react-native';

export default class UnixGroup extends Component {
  constructor(props){
    super(props);
  }
  render() {
    return (
      <View>
        <Text style={styles.unixGroupStyle}>
          {this.props.details.Name}
        </Text>
      </View>
    );
  }

}

const styles = StyleSheet.create({

 unixGroupStyle: {
   color: '#511',
   fontSize: 16,
   textAlign: 'center',
   marginBottom: 1,
   //backgroundColor: 'steelblue',
 },
 
});
