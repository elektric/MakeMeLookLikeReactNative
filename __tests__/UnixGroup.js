import 'react-native';
import React from 'react';
import UnixGroup from '../src/UnixGroup';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

test('UnixGroup renders', () => {
  let ug = {'Name':'UnixGroup2'};
  const tree = renderer.create(
    <UnixGroup details={ug} />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
