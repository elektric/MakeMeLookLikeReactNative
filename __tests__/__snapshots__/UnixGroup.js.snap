// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UnixGroup renders 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "flexDirection": "row",
      "justifyContent": "space-around",
    }
  }
>
  <View
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#511",
          "fontSize": 16,
          "marginBottom": 1,
          "textAlign": "center",
        }
      }
    >
      UnixGroup2
    </Text>
  </View>
  <View
    style={Object {}}
  >
    <RCTSwitch
      disabled={false}
      onChange={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      onValueChange={[Function]}
      style={
        Array [
          Object {
            "height": 31,
            "width": 51,
          },
          Object {},
        ]
      }
      value={true}
    />
  </View>
</View>
`;
